development/openstack-base-focal-wallaby/README.md

```diff
@@ -1,269 +1,371 @@
+Create a Focal amd64 instance called 'focal-1':
 
-then attach it to the instance we just booted:
+    openstack server create --image focal-amd64 --flavor m1.tiny \
+       --key-name mykey --network int_net \
+        focal-1
 
-    openstack server add volume focal-test <name-of-volume>
+## Assign a floating IP address
 
-The attached volume will be accessible once you login to the instance (see below).  It will need to be formatted and mounted!
+Request and assign a floating IP address to the new instance:
 
-### Accessing your instance
+    FLOATING_IP=$(openstack floating ip create -f value -c floating_ip_address ext_net)
+    openstack server add floating ip focal-1 $FLOATING_IP
 
-In order to access the instance you just booted on the cloud, you'll need to
-assign a floating IP address to the instance:
+## Log in to an instance
 
-    FIP=$(openstack floating ip create -f value -c floating_ip_address ext_net)
-    openstack server add floating ip focal-test $FIP
+Log in to the new instance:
 
-and then allow access via SSH (and ping) - you only need to do these steps
-once:
+    ssh -i ~/cloud-keys/id_mykey ubuntu@$FLOATING_IP
 
-    PROJECT_ID=$(openstack project list -f value -c ID \
-	       --domain admin_domain)
+The below commands are a good start to troubleshooting if something goes wrong:
 
-    SECGRP_ID=$(openstack security group list --project $PROJECT_ID \
-        | awk '/default/{print$2}')
+    openstack console log show focal-1
+    openstack server show focal-1
 
-    openstack security group rule create $SECGRP_ID \
-        --protocol icmp --ingress --ethertype IPv4
+## Access the cloud dashboard
 
-    openstack security group rule create $SECGRP_ID \
-        --protocol icmp --ingress --ethertype IPv6
+To access the dashboard (Horizon) first obtain its IP address:
 
-    openstack security group rule create $SECGRP_ID \
-        --protocol tcp --ingress --ethertype IPv4 --dst-port 22
+    juju status --format=yaml openstack-dashboard | grep public-address | awk '{print $2}' | head -1
 
-    openstack security group rule create $SECGRP_ID \
-        --protocol tcp --ingress --ethertype IPv6 --dst-port 22
+The password can be queried from Keystone:
 
-After running these commands you should be able to access the instance:
+    juju run --unit keystone/leader leader-get admin_passwd
 
-    ssh ubuntu@$FIP
+In this example, the address is '10.5.0.30' and the password is
+'eifah8Ixie3Aegee'.
 
-### Logging in to the OpenStack Dashboard
+The dashboard URL then becomes:
 
-First determine the IP address of the OpenStack Dashboard:
+**http://10.5.0.30/horizon**
 
-    juju status openstack-dashboard
+The final credentials needed to log in are:
 
-Type in the following URL in your web browser: https://DASHBOARD-IP/horizon/
+<!-- There are two spaces at the end of the next two lines -->
 
-To print your credentials:
+User Name: **admin**  
+Password: **eifah8Ixie3Aegee**  
```

Comment:
I know it is silly but I would put a place holder here rather than an example password.